scipy/sparse/linalg/eigen/_svds.py

```diff
@@ -58,42 +58,38 @@ def svds(A, k=6, ncv=None, tol=0, which='LM', v0=None,
 
     Parameters
     ----------
-    A : {sparse matrix, LinearOperator}
+    A : sparse matrix or LinearOperator
```

Comment:
I saw precedent for this. 
As I understand it, `{ }` indicate an enum, essentially: literal values that can be used. A user should not pass `LinearOperator` for `A`; they need to pass an instance of a `LinearOperator`.